src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -81,14 +81,18 @@
     <div class="search-button__overlay"></div>
     <div class="search-button__search-container">{% include "../components/search-field.html" %}</div>
   </div>
+
+  <header>
   {%- if theme_announcement -%}
     {% include "sections/announcement.html" %}
   {%- endif %}
   {% block docs_navbar %}
-    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
+    <div class="bd-header navbar navbar-expand-lg bd-navbar">
```

Comment:
As an aside, there's something that feels not quite right to me about putting external links under a heading that says "Site Navigation" but maybe that's an issue for another day.

Reply:
> As an aside, there's something that feels not quite right to me about putting external links under a heading that says "Site Navigation" but maybe that's an issue for another day.

yes let's punt on that.  Users like to do it (have external links in the main topbar nav) and would be mad if we removed the ability to do that.

Reply:
Right. But maybe the heading should be "Main Navigation" or something like that instead of "Site Navigation"? Or maybe we get rid of the visible heading altogether and set aria-label="Main" on the nav tag.

Reply:
I'm OK with "Main navigation" instead of "Site navigation"

Reply:
After thinking about this some more, may I make an argument to simply remove the heading?

1. Consistency between mobile and desktop, because this heading is not shown on desktop, but is shown on mobile
2. Frees up valuable screen real estate
3. Reduces number of strings to translate
4. Solves the problem of screen reader noise: "Site Navigation navigation" or "navigation Site navigation"
5. I'm not sure it's really needed in the mobile sidebar as the row of icon links followed by a horizontal line already creates strong visual break between the two nav sections: <br><img width="288" alt="icons and horizontal line separate site from section nav" src="https://github.com/pydata/pydata-sphinx-theme/assets/317883/e3631925-af18-4e6d-ac2b-2de8386f9f48">

Reply:
can you open a new issue to discuss further the idea of removing the "Site navigation" title?

Reply:
https://github.com/pydata/pydata-sphinx-theme/issues/1588 :)